src/migrations/20190115132117-create-dummy-hotel.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('DummyHotels', {
+      id: {
```

Comment:
Change this to es6 methods here is an article that will help.
[https://codewithhugo.com/using-es6-classes-for-sequelize-4-models/](here)

Reply:
I'm not sure you can convert the migrations to es6. I only converted the models to es6 methods